packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -364,6 +362,116 @@ export function getMiniflareOptions(
 	};
 }
 
+export function getPreviewMiniflareOptions(
+	normalizedPluginConfig: NormalizedPluginConfig,
+	viteConfig: vite.ResolvedConfig,
+): MiniflareOptions {
+	const entryWorkerConfig = normalizedPluginConfig.entryWorkerName
+		? normalizedPluginConfig.workers[normalizedPluginConfig.entryWorkerName]
+		: undefined;
+
+	const assetsDirectory = path.resolve(
+		viteConfig.root,
+		viteConfig.build.outDir,
+		'client',
+	);
+	const hasAssets = fs.existsSync(assetsDirectory);
+	const assets = hasAssets
```

Comment:
Nit: I think a name like `assetsOptions` or something like that would be clearer here